lib/samples/display_annotation/display_annotation.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import 'package:arcgis_maps/arcgis_maps.dart';
+import 'package:flutter/material.dart';
+
+void main() async {
+  // Supply your apiKey using the --dart-define-from-file command line argument
+  ArcGISEnvironment.apiKey = const String.fromEnvironment('API_KEY');
+
+  runApp(
+    const MaterialApp(
+      home: DisplayAnnotation(),
+    ),
+  );
+}
+
+class DisplayAnnotation extends StatefulWidget {
+  const DisplayAnnotation({super.key});
+
+  @override
+  State<DisplayAnnotation> createState() => _DisplayAnnotationState();
+}
+
+class _DisplayAnnotationState extends State<DisplayAnnotation> {
```

Comment:
```suggestion
class _DisplayAnnotationState extends State<DisplayAnnotation> with SampleStateSupport {
```
Hopefully sprint x's and more hands on deck we can get more scripted and/or github checks for us in samples.